pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -187,10 +187,13 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 		if customCommand.Stream {
 			cmdObj.StreamOutput()
 		}
-		err := cmdObj.Run()
+		output, err := cmdObj.RunWithOutput()
 		if err != nil {
 			return self.c.Error(err)
 		}
+		if customCommand.ShowOutput {
+			return self.c.Alert(cmdStr, output)
```

Comment:
Does this call `self.c.Refresh` somewhere in the background, because other control paths do invoke it (e.g. `return self.c.RunSubprocessAndRefresh(cmdObj)`)

Reply:
Not sure, I'll check